docs/release.md

```diff
@@ -41,10 +41,10 @@ imgtool is released through pypi.org (The Python package index) and
 requires that its version to be updated by editing
 `scripts/imgtool/__init__.py` and modifying the exported version:
 
-`imgtool_version = "X.Y.ZrcN"`
+`imgtool_version = "A.B.CrcN"`
 
-where `rcX`, `aX` and `bX` are accepted pre-release versions (just
-numbers for final releases). For more info see:
+where `rcN` is accepted for pre-release version under test, and numbers
+only for final releases. For more info see:
```

Comment:
This description is good, but I wonder if we should be clearer about keeping the mcuboot and imgtool versions in sync.

Reply:
Done.